lib/sync/sync-server.js

```diff
@@ -153,15 +153,10 @@ function stopAllWorkers(workers, cb) {
 function start(cb) {
   if (arguments.length < 1) throw new Error('start requires 1 argument');
 
-  if (syncStarted) {
-    return cb();
-  } else {
-    //prevent `start` getting called multiple times
-    syncStarted = true;
-  }
+  syncStarted = true;
```

Comment:
@wei-lee actually I'll keep this flag as its being checked when 'stop' is called and behaves differently if sync was already started.